Resources/Prototypes/_LostParadise/Entities/Objects/Decoration/halloween.yml

```
@@ -0,0 +1,34 @@
+    available:
+      - enum.DamageStateVisualLayers.Base:
+          spider_big: ""
+          spider_big2: ""
+          spider_mini1: ""
+          spider_mini2: ""
+          spider_mini3: ""
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)